.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Gradle Plugin Publish
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish plugin to gradle plugins portal
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: |
+            8
+            17
+          architecture: x64
+
+      - name: Show version number
+        id: version
+        run: |
+          echo "Version=${{ github.event.release.tag_name }}"
+
+      - name: Publish plugins with gradle
+        env:
+          GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+          GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+          PLUGIN_VERSION: ${{ github.event.release.tag_name }}
+        run: ./gradlew publishPlugins -Pgradle.publish.key=$GRADLE_PUBLISH_KEY -Pgradle.publish.secret=$GRADLE_PUBLISH_SECRET
```

Comment:
Q: Why do we set `GRADLE_PUBLISH_KEY` to both `env` and `-P` property?
If the secrets contain white spaces, the command results in an unexpected behavior. 
https://github.com/line/armeria/blob/6f16ffb9c62d267c648edb59166259ef3bc4775e/.github/workflows/publish-release.yml#L38-L40

Reply:
It would be better to use environment variables for future-proof.

Reply:
ah, just copy paste from other project like https://github.com/line/gradle-multi-project-support/blob/master/.github/workflows/publish.yml

looks like I can just do 
```
```

Reply:
Curious how we can use env for `gradle.publish.key`. Or read env at script?(May need to try)